README.md

```diff
@@ -30,12 +46,22 @@ To get a clone of the project, run `git clone https://github.com/white3d/Rails-R
 cd /Rails-React-Doctor-Appointment-App
 ```
 
+```
+rake db:migrate 
+```
+
+```
+rake start
```

Comment:
- [ ] Before the "rails db:migrate", your users will need to run "rails db:create" to create the databases.  Kindly add that to your instructions.